<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HV_isegSHQ</class>
 <widget class="QWidget" name="HV_isegSHQ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HV_isegSHQ</string>
  </property>
  <widget class="QGroupBox" name="CH1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="title">
    <string>CH1</string>
   </property>
   <widget class="QLabel" name="Label_V1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_I1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
    <property name="text">
     <string>I1:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Voltage1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Current1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_DesVoltage1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Voltage :</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DesVoltage1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <double>299.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="SetVoltage1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DesRamp1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <double>299.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="Label_DesRamp1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Ramp :</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SetRamp1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetVoltage1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetRamp1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Disable1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="State1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="CH2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="title">
    <string>CH2</string>
   </property>
   <widget class="QLabel" name="Label_V2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_I2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
    <property name="text">
     <string>I2:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Voltage2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Current2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_DesVoltage2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Voltage :</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DesVoltage2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <double>299.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="SetVoltage2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DesRamp2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <double>299.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="Label_DesRamp2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Ramp :</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SetRamp2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetVoltage2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetRamp2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Disable2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="State2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
